src/main/java/com/example/ajaxonboardingproject/controller/NatsCinemaHallController.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.ajaxonboardingproject.controller
+
+import CinemaHallOuterClass
+import ListOfCinemaHallsOuterClass
+import com.example.ajaxonboardingproject.model.CinemaHall
+import com.example.ajaxonboardingproject.service.CinemaHallService
+import com.example.ajaxonboardingproject.service.proto.converter.CinemaHallConverter
+import com.google.protobuf.Parser
+import io.nats.client.Connection
+import io.nats.client.Message
+import org.springframework.stereotype.Component
+
+@Component
+class NatsCinemaHallAddController(
+    private val converter: CinemaHallConverter,
+    private val service: CinemaHallService,
+    override val connection: Connection
+) : NatsController<CinemaHallOuterClass.CinemaHall> {
+    override val subject: String = "cinemaHall.add"
```

Comment:
consider extracting all subjects to one class called NatsSubject and put it in the same module with proto files (when you'll have a separate module for them). 
The goal is that other services can add to dependencies the module with API, and it will have info about what models should it use and to which subject it should be sent 

checkout our conventions for API. 
https://ajaxsystems.atlassian.net/wiki/spaces/Tech/pages/3676078201/Internal+API+conventions+v2

Implementing all conventions will be overkill for test project, pay attention to these topics
<img width="433" alt="image" src="https://github.com/OlehSarapuk/ajax-onboarding-project/assets/77619477/4e6e4a4f-06e0-48fc-8526-048fc42e71b9">

Reply:
Done!